### PR TITLE
[hi] Localize container-runtime-interface.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/container-runtime-interface.md` to Hindi `content/hi/docs/reference/glossary/container-runtime-interface.md`

Fixes :- #35600

### CODE REVIEW

**Comment by bishal7679** [comment]
/assign @anubha-v-ardhan Please leave some suggestions

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by bishal7679** [comment]
/assign @anubha-v-ardhan

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/container-runtime-interface.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+  - cri
+---
+
+क्यूबलेट और कंटेनर रनटाइम के बीच संचार के लिए मुख्य प्रोटोकॉल।
+
+<!--more-->
+
+कुबेरनेट्स कंटेनर रनटाइम इंटरफेस (सीआरआई) मुख्य [gRPC](https://grpc.io) प्रोटोकॉल के परिभाषित करता है [क्लस्टर घटक](/docs/concepts/overview/components/#node-components) {{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} और {{< glossary_tooltip text="कंटेनर रनटाइम" term_id="container-runtime" >}} के बीच संचार के लिए।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/container-runtime-interface.md:18]: While the machine translation sounded okay, I'd suggest that we write it this way to simplify what we're trying to convey. 
```suggestion
[क्लस्टर घटक](/docs/concepts/overview/components/#node-components) {{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} और {{< glossary_tooltip text="कंटेनर रनटाइम" term_id="container-runtime" >}} के बीच संचार के लिए कुबेरनेट्स कंटेनर रनटाइम इंटरफेस (सीआरआई) मुख्य [gRPC](https://grpc.io) प्रोटोकॉल को परिभाषित करता है।
```

**Review by divya-mohan0209 — COMMENTED** [review]
I think phrasing it differently makes more sense from a simplification perspective.

**Comment by divya-mohan0209** [comment]
@anubha-v-ardhan @Garima-Negi whenever you have the chance please give this a second pass.
/lgtm

**Pull request merged**